docs/reference/reference-training/training/api-pytorch-reference.rst

```diff
@@ -5,16 +5,25 @@
 +--------------------------------------------+
 | User Guide                                 |
 +============================================+
-| :doc:`/training/apis-howto/api-pytorch-ug` |
+| :ref:`pytorch_trial_ug`                    |
 +--------------------------------------------+
 
+Determined offers a PyTorch-based training loop that is fully integrated with the Determined
+platform which includes: - :class:`~determined.pytorch.PyTorchTrial`, which you must subclass to
```

Comment:
something went wrong when running rstfmt against the sample text I gave you, sorry.